### PR TITLE
Buff idleable quests

### PR DESCRIPTION
**Pull request by anapple96**
Received quite a bit of complains regarding QP

### CODE REVIEW

**Review by RedSparr0w — CHANGES_REQUESTED** [review]
Some of the numbers seem to be a little high, and some are being double/triple buffed due to relying on already buffed values.

File: src/scripts/GameConstants.ts [hunk 1]
```diff
@@ -158,10 +158,10 @@ namespace GameConstants {
     // Numbers calculated by Dimava assumes ability to 1 shot on high routes and some use oak items,
     //   which are now nerfed slightly until upgraded, so those numbers may need further adjusting
     const questBase = 1; // change this to scale all quest points
-    export const DEFEAT_POKEMONS_BASE_REWARD  = questBase * 1;
-    export const CAPTURE_POKEMONS_BASE_REWARD = DEFEAT_POKEMONS_BASE_REWARD / 0.8; // Defeat reward divided by chance to catch (guessed)
-    export const GAIN_MONEY_BASE_REWARD       = questBase * 0.0017;  // Dimava
-    export const GAIN_TOKENS_BASE_REWARD      = CAPTURE_POKEMONS_BASE_REWARD / 13; // <route number> tokens gained for every capture
+    export const DEFEAT_POKEMONS_BASE_REWARD  = questBase * 3;
```
[the remark this answers is not in the document]
Comment (anapple96) [reply src/scripts/GameConstants.ts:161]: Yeah, that wasn't intended, 9x buff is too high

File: src/scripts/GameConstants.ts [hunk 2]
```diff
@@ -158,10 +158,10 @@ namespace GameConstants {
...
+    export const CAPTURE_POKEMONS_BASE_REWARD = DEFEAT_POKEMONS_BASE_REWARD / 0.5; // Defeat reward divided by chance to catch (guessed)
```
[the remark this answers is not in the document]
Comment (anapple96) [reply src/scripts/GameConstants.ts:162]: Nerfed defeat base

File: src/scripts/GameConstants.ts [hunk 3]
```diff
@@ -158,10 +158,10 @@ namespace GameConstants {
...
+    export const GAIN_MONEY_BASE_REWARD       = questBase * 0.005;  // Dimava
```
[the remark this answers is not in the document]
Comment (anapple96) [reply src/scripts/GameConstants.ts:163]: Nerfed it a bit

File: src/scripts/GameConstants.ts [hunk 4]
```diff
@@ -158,10 +158,10 @@ namespace GameConstants {
...
+    export const GAIN_TOKENS_BASE_REWARD      = CAPTURE_POKEMONS_BASE_REWARD / 9; // <route number> tokens gained for every capture
```
[the remark this answers is not in the document]
Comment (RedSparr0w) [reply src/scripts/GameConstants.ts:164]: This one is still nearly 4x as much as it previously was.

**Pull request closed without merging**